healthcare/healthcare/doctype/patient_appointment/test_patient_appointment.py

```
@@ -838,7 +838,15 @@ def test_department_match_blocks_even_if_service_unit_differs(self):
 
 
 def create_practitioner_availability(
-	start, end, scope=None, scope_type=None, date=None, type="Available", service_unit=None
+	start,
+	end,
+	scope=None,
+	scope_type=None,
+	date=None,
+	type="Available",
```

Ruff (0.15.17)

[error] Function argument `type` is shadowing a Python builtin

(A002)